pep/app/styles/_variables-theme.scss

```diff
@@ -30,3 +31,4 @@ $headings-font-family: Muli, -apple-system, BlinkMacSystemFont, 'Segoe UI', Robo
 $hr-border-color: rgba(#000, 0.06);
 $dropdown-border-color: #dbdbdb;
 $dropdown-border-radius: 0.21rem;
+$table-border: #e9e9e9;
```

Comment:
shouldn't need to redefine `$white` as it already exists, and `$table-border` would also need to be defined in `styles/themes/tomato.scss` too, otherwise the sass build will fail. Although I think we should probably remove this variable altogether, and just use the existing bootstrap `$table-border-color` variable where needed in _ember-table.scss.

Reply:
looks like `$table-border` is no longer being used anywhere